pom.xml

```diff
@@ -86,6 +86,12 @@
         <air.test.thread-count>2</air.test.thread-count>
         <!-- Be conservative about memory allotment, because tests start background process (e.g. docker containers) -->
         <air.test.jvmsize>3g</air.test.jvmsize>
+        <!-- G1 default region size for a small heap is 1MB. Tests (TestHiveConnectorTest.testMultipleWritersWhenTaskScaleWritersIsEnabled) in particular
+             were observed allocating large number of byte[] of about 1.6MB in size, thus being treated as humongous allocations and preventing heap saturation.
+             Force bigger region size in attempt to help G1 utilize heap fully. -->
+        <air.test.jvm.additional-arguments>
+            -XX:G1HeapRegionSize=32M
```

Comment:
Why 32? Is this some sort of standard for Trino deployments?

Reply:
this is the max value.

also, we use `-XX:G1HeapRegionSize=32M` in product tests already

no other special meaning.

Reply:
32M is also recommended Trino default

Reply:
Trino was actually designed with the assumption that the JVM is run with 32M. That's related to what the JVM considers a "humongous allocation", which get special treatment and can cause issues for the GC. Many of the heuristics in Trino take into account that number as a threshold to avoid humongous allocations.